src/Microsoft.Health.Dicom.Blob/Features/Export/AzureBlobExportSink.cs

```diff
@@ -63,8 +63,8 @@ public async Task<bool> CopyAsync(ReadResult value, CancellationToken cancellati
             return false;
         }
 
-        using Stream sourceStream = await _source.GetStreamingFileAsync(value.Identifier.Version, value.Identifier.Partition.Name, cancellationToken);
-        BlobClient destBlob = _dest.GetBlobClient(_output.GetFilePath(value.Identifier));
+        using Stream sourceStream = await _source.GetStreamingFileAsync(value.Instance.VersionedInstanceIdentifier.Version, value.Instance.VersionedInstanceIdentifier.Partition, value.Instance.InstanceProperties.fileProperties, cancellationToken);
```

Comment:
If this call returns `412 Precondition Failed`, then we should probably also invoke the `CopyFailure` event and enqueue an error. It is a bit strange that we'll have the Export API available for "IDP" as the storage is already exposed, but we can gracefully deprecate as indexing and IDP come online. Or maybe we'll deprecate it before GA

Reply:
Can you add a test for this too?